accelerate/nextjs-starter/lib/utils/query.ts

```diff
@@ -0,0 +1,30 @@
+import prisma from "../db";
+import { CacheStrategy } from "../types";
+
+export const getQuotes = async (strategy?: CacheStrategy) => {
+  const start = Date.now();
+
+  const result = await prisma.quotes
+    .findMany({
+      cacheStrategy: strategy,
+      orderBy: {
+        id: "desc",
+      },
+      take: 1,
+    })
+    .withAccelerateInfo();
```

Comment:
most of this code is self documenting, but we might want to link to documentation with this line.